sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -0,0 +1,3175 @@
MgMzIyLjg2MyAyMDAuNTI2WiIgZmlsbD0iIzQwN0JGRiIvPgo8cGF0aCBkPSJNMzI1LjA2OSAyMDAuMDhDMzI1LjA1MiAyMDAuMDg2IDMyNS4wMzMgMjAwLjA4NiAzMjUuMDE2IDIwMC4wOEMzMjQuNDEzIDIwMC4xNDggMzIyLjg5NiAxOTkuODM1IDMyMi42MzcgMTk5LjMzN0MzMjIuNTk4IDE5OS4yNjEgMzIyLjU4OCAxOTkuMTc1IDMyMi42MSAxOTkuMDkzQzMyMi42MzIgMTk5LjAxMSAzMjIuNjg0IDE5OC45MzkgMzIyLjc1NyAxOTguODg5QzMyMi44MjUgMTk4LjgzMSAzMjIuOTA2IDE5OC43ODYgMzIyLjk5MyAxOTguNzU5QzMyMy4wODEgMTk4LjczMSAzMjMuMTc0IDE5OC43MjEgMzIzLjI2NyAxOTguNzI5QzMyNC4wNDIgMTk4Ljc4NCAzMjUuMDQyIDE5OS44NTMgMzI1LjA4OSAxOTkuODk2QzMyNS4xIDE5OS45MTQgMzI1LjEwNSAxOTkuOTM0IDMyNS4xMDUgMTk5Ljk1NEMzMjUuMTA1IDE5OS45NzUgMzI1LjEgMTk5Ljk5NSAzMjUuMDg5IDIwMC4wMTNDMzI1LjA5MSAyMDAuMDM3IDMyNS4wODQgMjAwLjA2MSAzMjUuMDY5IDIwMC4wOFpNMzIyLjk2OSAxOTkuMDQySDMyMi45MjlDMzIyLjgxIDE5OS4xNDcgMzIyLjg0MyAxOTkuMjA4IDMyMi44NTYgMTk5LjIzM0MzMjMuMDA4IDE5OS41MjggMzI0LjA1NSAxOTkuODQ3IDMyNC43MjQgMTk5Ljg0N0MzMjQuMzQxIDE5OS40MTkgMzIzLjgzMyAxOTkuMTA0IDMyMy4yNiAxOTguOTM4QzMyMy4xNTIgMTk4LjkzNiAzMjMuMDQ4IDE5OC45NzQgMzIyLjk2OSAxOTkuMDQyWiIgZmlsbD0iIzQwN0JGRiIvPgo8cGF0aCBkPSJNMzA0LjMxOSA2MS40MDYxTDMwNS45NDkgNjMuMjQ4NEMzMDYuNDk4IDYzLjg2MjUgMzA3LjA0MiA2NC41MDczIDMwNy41NzggNjUuMTUyMUMzMDguNjUxIDY2LjQyOTQgMzA5LjY5OCA2Ny43MjMxIDMxMC43MTggNjkuMDMzMkMzMTEuNzE5IDcwLjM0NzQgMzEyLjcwNiA3MS42NzM4IDMxMy41OTMgNzMuMDEyNkMzMTQuNDgxIDc0LjM1MTMgMzE1LjMzNiA3NS43MTQ2IDMxNi4wMzEgNzcuMDc3OUwzMTYuMTcgNzcuMzM1OUwzMTYuMjIzIDc3LjQ1MjVDMzE2LjI3MyA3Ny41NDQ5IDMxNi4zMTMgNzcuNjQxNiAzMTYuMzQzIDc3Ljc0MTJDMzE2LjQzMyA3OC4wMTkxIDMxNi41MDQgNzguMzAyMiAzMTYuNTU1IDc4LjU4ODZDMzE2LjY4MyA3OS4yOTM1IDMxNi43NjMgODAuMDA1MyAzMTYuNzkzIDgwLjcxOTZDMzE2Ljg1OSA4Mi4yMzY0IDMxNi44NCA4My44NTE1IDMxNi43NDcgODUuNDcyN0MzMTYuNTc1IDg4LjcxNTEgMzE2LjI1NyA5Mi4wNjgxIDMxNS45MDUgOTUuMzU5N0wzMTkuNjk1IDk2LjE3NjVDMzIwLjk0NSA5Mi45MDAyIDMyMS45MzEgODkuNTQxOSAzMjIuNjQzIDg2LjEyOThDMzIzLjAxIDg0LjM2NjcgMzIzLjI2MiA4Mi41ODQ2IDMyMy4zOTggODAuNzkzMkMzMjMuNDYxIDc5LjgxOTcgMzIzLjQ2MSA3OC44NDM2IDMyMy4zOTggNzcuODcwMUMzMjMuMzUgNzcuMzE3NiAzMjMuMjY5IDc2Ljc2ODEgMzIzLjE1MyA3Ni4yMjQzQzMyMy4wNzMgNzUuOTIzNCAzMjMuMDE0IDc1LjY0MDkgMzIyLjg4OCA3NS4yOTdMMzIyLjcxNiA3NC44MThMMzIyLjU3NyA3NC40OTg3QzMyMS4xMzkgNzEuMTQ3MyAzMTkuMzIyIDY3Ljk0NjMgMzE3LjE1OCA2NC45NDk0QzMxNi4xMTEgNjMuNDc1NiAzMTUuMDExIDYyLjA1MDkgMzEzLjg0NSA2MC42NTA3QzMxMy4yNjkgNTkuOTU2OCAzMTIuNjc5IDU5LjI3NTIgMzEyLjA3NiA1OC41OTM1QzMxMS40NzQgNTcuOTExOSAzMTAuODc3IDU3LjI2NyAzMTAuMTg4IDU2LjU1NDdMMzA0LjMxOSA2MS40MDYxWiIgZmlsbD0iI0ZGQjU3MyIvPgo8cGF0aCBkPSJNMzA5LjIwMiA1NC4wNDRDMzE0LjEzIDU0LjY5NDkgMzE5LjQ3NyA2NS40NTM5IDMxOS40NzcgNjUuNDUzOUwzMTAuMTQ5IDczLjYzMzhDMzEwLjE0OSA3My42MzM4IDMwNi4yOCA2OC4yMjM1IDMwMy4wOTQgNjIuOTczQzI5OS42MjkgNTcuMjU1NyAzMDMuMjM5IDUzLjI1NzkgMzA5LjIwMiA1NC4wNDRaIiBmaWxsPSIjNDA3QkZGIi8+CjxwYXRoIGQ9Ik0zMTYuMzU2IDk0LjMyMDNMMzEzLjIzNiA5OC4xNDYyTDMxOS43MjEgOTkuNjAxNkMzMTkuNzIxIDk5LjYwMTYgMzIxLjU3NiA5NS43NzU3IDMxOS40NTcgOTQuNTY1OUwzMTYuMzU2IDk0LjMyMDNaIiBmaWxsPSIjRkZCNTczIi8+CjxwYXRoIGQ9Ik0zMTMuMzQzIDEwMi4xNTlMMzE4Ljc1NSAxMDMuMTI5TDMxOS43MjMgOTkuNjAzOUwzMTMuMjM3IDk4LjE0ODRMMzEzLjM0MyAxMDIuMTU5WiIgZmlsbD0iI0ZGQjU3MyIvPgo8L3N2Zz4K\"></figure>",
```

Comment:
I would say that this will increase the content of the json payload to load the form definition and this is cached only daily. I would say that this is a wonderful solution for the future if some change is needed, but now I would prefer having this image in Vue source in the same way the student application is doing.
@ann-aot @guru-aot @dheepak-aot thoughts?

Reply:
actually, I kind of like this , as you said, it gives the user flexibility to change without code change.

Reply:
So, is it better to have the images being loaded every time as a part of a big request from the server every time?
I would say that this should be used as a second option, IMO.

Reply:
Considering that the image loads only once daily and the image is considerably light, I would say benefits outweigh the risk. So option one IMO.

Reply:
I feel like having this image in Vue source, like the previous screens, to be consistent.

Reply:
maybe we can have a quick up and finalize this after the demo, maybe I am missing something

Reply:
@dheepak-aot, when I said that the page loads once a day I was just being too right level.
The form definition (right now with 310Kb) will load every time that a user loads the Institution Portal in a new Tab The form size will grow more if more images are embedded.
From the user perspective there will no much impact. From our server perspective it will be downloading every time the images because they are embedded. The form without the images is something around 126KB, 40% of the size.

Lets say that this is not a Institution User portal, are we going to use embedded images for the welcome page on Student Portal, for instance? @dheepak-aot @ann-aot 

The ideal scenario, IMO, would be the Ministry offer a place that we can dump the images and just add the reference to the image in the form.io definition. Since we do not have it right now, we were adding to Vue source.